File: projects/m1/034-integer-to-ordinal/valentinaBazzacchi/solution-integer-to-ordinal.js
function getOrdinalNumber(integer) {
    let ordinalNumbers = ["first", "second", "third", "fourth", "fifth", "sixth", "seventh", "eighth", "ninth", "tenth", "eleventh", "twelfth"];
    ordinalNumbers = ordinalNumbers[integer-1];
    if (integer >= 1 && integer <= 12) {  
        return integer + " - " + ordinalNumbers;
    } else {
        return "" 
    }
}
   
let number = prompt("Enter a integer from 1 to 12");
alert(getOrdinalNumber(number));

////////////////////////////////////////////////////////////////////

function getOrdinalNumber(integer) {
    let ordinalNumbers = ["first", "second", "third", "fourth", "fifth", "sixth", "seventh", "eighth", "ninth", "tenth", "eleventh", "twelfth"];
    ordinalNumbers = ordinalNumbers[integer-1];
    return ordinalNumbers || ""; // Used logical operator OR, if ordinalNumbers is false return a blank line 
}

let number = prompt("Enter a integer from 1 to 12");
alert(getOrdinalNumber(number));

File: projects/m1/034-integer-to-ordinal/solutions/jimmyhu/solution-integer-to-ordinal.py

number_input = int(input('Enter the number: '))
ordinals = ['first','second','third','fourth','fifth','sixth','seventh','eighth','nineth','tenth','eleventh','twelfth']
def poistion(number,array):
    if number >= 1 and number <= 12:
        return array[number -1]
    else:
        return ' '
print(poistion(number_input,ordinals))

if __name__ == '__main__':
    def test():
        for i in range(1,13):
            print(f"{i} " + poistion(i,ordinals))
    test()
